systemvm/debian/etc/systemd/system/cloud-postinit.service

```diff
@@ -1,7 +1,7 @@
 [Unit]
 Description=CloudStack post-patching init script
 After=cloud-early-config.service network.target local-fs.target
-Before=ssh.service
+#Before=ssh.service
```

Comment:
remove commented code if not used, this may be useful if the systemvmtemplate is already say patched once and sshd is already enabled - pl check.